hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/BlockDeletingService.java

```diff
@@ -150,8 +153,15 @@ public BackgroundTaskQueue getTasks() {
         queue.add(containerBlockInfos);
         totalBlocks += containerBlockInfo.getNumBlocksToDelete();
       }
+      metrics.setDurationOfLastIteration(Time.monotonicNow() - startTime);
+      metrics.setStartTimeOfLastIteration(startTime);
+      metrics.setBlockChosenCountInLastIteration(totalBlocks);
+      metrics.setContainerChosenCountInLastIteration(containers.size());
       metrics.incrTotalBlockChosenCount(totalBlocks);
       metrics.incrTotalContainerChosenCount(containers.size());
+      metrics.incrSuccessCountInLastIteration(BlockDeletingTask.getSuccessBlockDeleteInLastIteration());
+      metrics.incrSuccessBytesInLastIteration(BlockDeletingTask.getSuccessBytesDeletedInLastIteration());
+      metrics.incrFailureCountInLastIteration(BlockDeletingTask.getFailBlockDeleteInLastIteration());
```

Comment:
Why are we using incr here, when metric is just for last iteration?

Reply:
I have Fixed it now, thanks for catching this